domains/zyrouge.json

```diff
@@ -5,6 +5,6 @@
     "twitter": "_zyrouge_"
   },
   "record": {
-    "CNAME": "zyrouge-is-a-dev.pages.dev"
+    "CNAME": "zyrouge.github.io"
```

Comment:
you're going to have to rename https://github.com/zyrouge/zyrouge.is-a.dev to `zyrouge/zyrouge.github.io`

Reply:
Not really, I don't really want to block `zyrouge.github.io` to this site since I have sites like `https://zyrouge.github.io/symphony`.

Reply:
@TheBotlyNoob It will still work, even with the repo named that.